.gitlab.mk

```diff
@@ -128,11 +128,14 @@ package: deploy_prepare
 
 deploy:
 	echo ${GPG_SECRET_KEY} | base64 -d | gpg --batch --import || true
-	./tools/update_repo.sh -o=${OS} -d=${DIST} \
-		-b="${LIVE_REPO_S3_DIR}/${BUCKET}" build
-	if [ "${CI_COMMIT_TAG}" != "" ]; then \
+	# found that libcreaterepo_c.so installed in local lib path
+	export LD_LIBRARY_PATH=/usr/local/lib ; \
```

Comment:
How about something like [this](https://stackoverflow.com/a/49524393/1598057)? Or, better, setup the variable in the action, where we build the tool?

Reply:
I've tried to set it to action, but it failed work for update_repo tool. But using your example I've successfully checked it using export as target:
https://github.com/tarantool/tarantool/runs/2288050206#step:5:13393

Reply:
> I've tried to set it in action, but it failed to work for update_repo tool.

Via $GITHUB_ENV?